src/main/java/space/obminyashka/items_exchange/exception/EmailTokenNotExistsException.java

```diff
@@ -0,0 +1,11 @@
+package space.obminyashka.items_exchange.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class EmailTokenNotExistsException extends Exception {
```

Comment:
You can make it consistent and rename it to EmailValidationCodeNotFound or simply CodeNotFound exception